Script/AtomicEditor/ui/modal/build/platforms/AndroidSettingsWidget.ts

```diff
@@ -125,6 +125,8 @@ class AndroidSettingsWidget extends Atomic.UIWidget implements BuildSettingsWind
 
     refreshAndroidTargets() {
 
+        if (ToolCore.toolEnvironment.toolPrefs.androidSDKPath.length <= 0) return;
```

Comment:
Wouldn't it be better to check if the path exists and possibly report an error (at least to the console, using LOGERRORF) ?  I don't think that check should be here, more like in the native platform.refreshAndroidTargets